app/views/pages/changelog.html.erb

```diff
@@ -4,9 +4,25 @@
 <div class="space-y-4">
   <h1 class="text-gray-900 text-xl font-medium mb-4">What's New</h1>
```

Comment:
I know other spots in the codebase are still lacking this, but we should probably include an i18n `t(".title")` here

Reply:
Sure, I updated it!